### ISSUE TITLE
Target.query is not a function

### ISSUE THREAD

**Issue by Zuluseq** [strapi/strapi#8700]
There is a problem with Media Library when click to Media Library menu or when I try upload the image file

### Steps to reproduce the behavior

1. Go to 'Media Library' or upload any file to gallery
2. See error

### System

Node.js version: 12.18
Database: postgresql
Operating system: Docker containers
Docker images version:  3.3.0

Bug logs:

```
	[2020-11-19T14:26:10.289Z] error TypeError: Target.query is not a function
	    at /srv/app/node_modules/bookshelf/lib/eager.js:62:23
	    at /srv/app/node_modules/bookshelf/node_modules/lodash/lodash.js:3543:27
	    at /srv/app/node_modules/bookshelf/node_modules/lodash/lodash.js:4925:15
	    at baseForOwn (/srv/app/node_modules/bookshelf/node_modules/lodash/lodash.js:2990:24)
	    at /srv/app/node_modules/bookshelf/node_modules/lodash/lodash.js:4894:18
	    at baseMap (/srv/app/node_modules/bookshelf/node_modules/lodash/lodash.js:3542:7)
	    at Function.map (/srv/app/node_modules/bookshelf/node_modules/lodash/lodash.js:9580:14)
	    at EagerRelation.morphToFetch (/srv/app/node_modules/bookshelf/lib/eager.js:56:9)
	    at EagerRelation.eagerFetch (/srv/app/node_modules/bookshelf/lib/eager.js:25:19)
	    at EagerRelation.<anonymous> (/srv/app/node_modules/bookshelf/lib/base/eager.js:67:14)
	From previous event:
	    at /srv/app/node_modules/bookshelf/lib/eager.js:96:73
	From previous event:
	    at EagerRelation._eagerLoadHelper (/srv/app/node_modules/bookshelf/lib/eager.js:83:23)
	    at Sync.<anonymous> (/srv/app/node_modules/bookshelf/lib/eager.js:31:31)
	From previous event:
	    at EagerRelation.eagerFetch (/srv/app/node_modules/bookshelf/lib/eager.js:31:8)
	    at EagerRelation.<anonymous> (/srv/app/node_modules/bookshelf/lib/base/eager.js:67:14)
	From previous event:
	    at Child._handleEager (/srv/app/node_modules/bookshelf/lib/collection.js:527:73)
	    at Child.<anonymous> (/srv/app/node_modules/bookshelf/lib/collection.js:179:27)
	    at processImmediate (internal/timers.js:456:21)
	From previous event:
	    at Child.<anonymous> (/srv/app/node_modules/bookshelf/lib/collection.js:177:12)
	From previous event:
	    at Child.fetchAll (/srv/app/node_modules/bookshelf/lib/model.js:878:10)
	    at Object.find (/srv/app/node_modules/strapi-connector-bookshelf/lib/queries.js:66:8)
	    at fn (/srv/app/node_modules/strapi-database/lib/queries/helpers.js:31:54)
	    at Object.find (/srv/app/node_modules/strapi-database/lib/queries/helpers.js:15:24)
	    at processTicksAndRejections (internal/process/task_queues.js:97:5)
	    at async find (/srv/app/node_modules/strapi-plugin-upload/controllers/upload/admin.js:39:19)
	    at async /srv/app/node_modules/strapi/lib/middlewares/router/utils/routerChecker.js:79:22
	[2020-11-19T14:26:10.290Z] debug GET /upload/files?_limit=10&_start=0&_sort=updated_at:DESC (91 ms) 500
	[2020-11-19T14:26:10.291Z] debug GET /admin/21b3848a32fce5b0f5014948186f6964.woff2 (17 ms) 200
	[2020-11-19T14:26:10.295Z] debug GET /admin/6d20cff5b3255dd0078f935c34e2b882.woff2 (16 ms) 200
	[2020-11-19T14:26:10.301Z] debug GET /upload/files/count? (88 ms) 200
```

**Comment by alexandrebodin** [comment]
Hi, this was fixed in versions > v3.3.1 :)

**Comment by alexandrebodin** [comment]
Closing as resolved
